test/sales_engine_test.rb

```diff
@@ -12,39 +12,14 @@ def test_response_to_startup
     assert sales_engine.respond_to?(:startup)
   end
 
-  def test_a_sales_engine_has_a_customer_repository
-    sales_engine.startup_customer
+  def test_a_sales_engine_has_repositories
```

Comment:
Assignment Branch Condition size for test_a_sales_engine_has_repositories is too high. [20/15]